### DISCUSSION TITLE
Increase the playlist limit

### DISCUSSION THREAD

**Discussion by GSculerlor** [jagrosh/MusicBot#616] [Questions]
Hello! I don't really know whether this should be here or should be on lavaplayer issue and I'm sure this question has already been asked many times but I can't find the answer. So I've been poking around on the source code to increase the limit of the playlist (currently it always capped at 100 items even the playlist had more than 100 items). I read on lavaplayer code that we can set the max items by changing `setPlaylistPageCount` value. I already tried to change it on `PlayerManager` but seems like it does nothing. Any workaround for this?

Thank you

**Comment by luxus** [comment]
there is a bug report for that.. https://github.com/jagrosh/MusicBot/issues/625
